use super::GlobalError;

pub fn run() -> Result<(), GlobalError> {
    Ok(())
}

#[test]
fn challenge_one_day_19() -> Result<(), GlobalError> {
    Ok(())
}
